ros_sgp_ipp/OfflineIPPData data
---
bool success